trackbs.go

```diff
@@ -163,7 +164,7 @@ func (tbs *TrackingBlockstore) Get(ctx context.Context, c cid.Cid) (blocks.Block
 	tbs.getCh <- c // TODO: should we be tracking all requests? or all successful servings?
 	blk, err := tbs.bs.Get(ctx, c)
 	if err != nil {
-		if xerrors.Is(err, blockstore.ErrNotFound) {
+		if blk == nil { // not found
```

Comment:
import `github.com/ipfs/go-ipld-formats`

and use `if errors.Is(err, ipld.ErrNotFound{}) {` instead.

Reply:
@alvin-reyes this is wrapped in.
You can do `if ipld.IsNotFound(err)`, no need to do `errors.Is` directly.